toucan_connectors/postgres/postgresql_connector.py

```diff
@@ -57,7 +57,11 @@ def _retrieve_data(self, data_source):
         connection = pgsql.connect(**self.get_connection_params(database=data_source.database))
 
         query_params = data_source.parameters or {}
-        df = pd.read_sql(data_source.query, con=connection, params=query_params)
+        df = pd.read_sql(
+            data_source.query,
+            con=connection,
+            params={k: (tuple(v) if isinstance(v, list) else v) for (k, v) in query_params.items()},
```

Comment:
Coud you maybe extract this `tuple(v) if isinstance(v, list) else v` into a small utility function (I suggest `adapt_param_type` or something like it ?) this will allow to have a nice docstring to document all cases where we have to alter the input variable (today we only the list=>tuple one, but we may have others in the future)